docs/examples/custom_operations/custom_operator/naive_histogram/CMakeLists.txt

```diff
@@ -0,0 +1,36 @@
+cmake_minimum_required(VERSION 3.18)
+set(CMAKE_CUDA_ARCHITECTURES "50;52;60;61;70;75;80;86")
```

Comment:
Is there a way for us to obtain this from DALI project definition? If this is hard to do or very convoluted, let's not go there.

Reply:
I'm not sure, what do you mean here?